automation/jenkins_build.sh

```diff
@@ -26,25 +27,36 @@ fi
 
 # Checkout meta-resin
 if [ "$metaResinBranch" == "__ignore__" ]; then
-    echo "INFO: Using the default meta-resin revision (as configured in submodules)."
+    echo "[INFO] Using the default meta-resin revision (as configured in submodules)."
 else
-    echo "INFO: Using special meta-resin revision from build params."
+    echo "[INFO] Using special meta-resin revision from build params."
     pushd $WORKSPACE/layers/meta-resin > /dev/null 2>&1
     git fetch --all
     git checkout --force origin/$metaResinBranch
     popd > /dev/null 2>&1
 fi
 
 # Run build
-$WORKSPACE/resin-yocto-scripts/build/barys \
-    --log \
-    --remove-build \
-    --machine "$MACHINE" \
-    --supervisor-tag "$supervisorTag" \
-    ${BARYS_ARGUMENTS_VAR} \
-    --shared-downloads "$JENKINS_DL_DIR" \
-    --shared-sstate "$JENKINS_SSTATE_DIR" \
-    --rm-work
+docker stop yocto-build 2> /dev/null || true
+docker rm yocto-build 2> /dev/null || true
```

Comment:
This should be `docker rm --volumes yocto-build 2> /dev/null || true` in order to make sure we clean up temporary volumes used by the container